### ISSUE TITLE
Add line numbers and highlighting to code snippets as comments

### ISSUE THREAD

**Issue by fhightower** [ThreatConnect-Inc/ThreatConnect_Developer_Docs#152]
### Description:
Something like:

```
.. code-block:: python
    :linenos:
    :lineno-start: 1
    :emphasize-lines: 14-16

    ...
```

(not sure if the `lineno-start` is needed)

- [x] Groups
- [x] Indicators
  - [x] Indicator metadata
- [x] Tags
- [x] Tasks
  - [x] Task metadata
- [x] Victims
  - [x] Victim metadata

**Comment by fhightower** [comment]
I've added line numbers to the python docs. Just need to go through and highlight the appropriate lines.

**Comment by fhightower** [comment]
Fixed with 31e53c3ece9f42ae00b7c912a70d2c33f70ab753
